erpnext/projects/doctype/task/task.py

```diff
@@ -318,6 +318,14 @@ def add_node():
 
 	if args.parent_task == 'All Tasks' or args.parent_task == args.project:
 		args.parent_task = None
+		if args.parent_task == args.project:
+			args.company == frappe.get_value('Project', args.project, 'company')
+		else:
+			from frappe.defaults import get_user_default
```

Comment:
Any specific reason for not making this global import at top?

Reply:
Also @ahmadpak can you please refresh the PR by rebasing/merging `develop`.

Reply:
No specific reason. I will do import to the top also rebase